scene/gui/text_edit.h

```diff
@@ -658,6 +658,7 @@ class TextEdit : public Control {
 
 #ifndef DISABLE_DEPRECATED
 	void _set_selection_mode_compat_86978(SelectionMode p_mode, int p_line = -1, int p_column = -1, int p_caret = 0);
+	String _get_line_column_at_pos_bind_compat_100913(const Vector2 &p_pos, bool p_allow_out_of_bounds = true) const;
```

Comment:
```suggestion
	Point2i _get_line_column_at_pos_bind_compat_100913(const Vector2 &p_pos, bool p_allow_out_of_bounds = true) const;
```
Same with the compat file.
I think this should fix the validation error.

Reply:
Thanks, changing the return type from `String` was part of it, but I was also missing a `DEFVAL`.